frontend/src/features/comparisonSeries/ComparisonSeries.tsx

```diff
@@ -16,6 +26,16 @@ interface Props {
   length: number;
 }
 
+async function getUserComparisons(
+  pollName: string
+): Promise<ComparisonModel[]> {
+  const comparisons = await UsersService.usersMeComparisonsList({
+    pollName,
+  });
```

Comment:
By default the number of comparisons is limited to 30 per page. 
100 should be enough, (66 pairs of candidates in this poll).
```suggestion
  const comparisons = await UsersService.usersMeComparisonsList({
    pollName,
    limit: 100,
  });
```